src/box/relay.cc

```diff
@@ -1155,6 +1155,8 @@ static void
 relay_send_initial_join_row(struct xstream *stream, struct xrow_header *row)
 {
 	struct relay *relay = container_of(stream, struct relay, stream);
+	if (iproto_type_is_raft_request(row->type))
+		return relay_send_raft(relay, row);
```

Comment:
Please, lets turn the `if (is raft request)` inside `relay_send_raft()` into an assertion. Otherwise it looks quite strange. When a request is not Raft, the function called "send raft" just ignores it. As a result, either the caller already knows it is a Raft request and uses this function, or has to check the type before calling the function. Either way the `if` inside of the function never fails. Which means it is an assertion. No?

Reply:
Agree, fixed